build.cmd

```diff
@@ -301,4 +311,5 @@ echo linuxmscorlib - Build mscorlib for Linux
 echo osxmscorlib - Build mscorlib for OS X
 echo skiptestbuild - Skip building tests
 echo toolset_dir - toolset directory -- Arm64 use only.
+echo Priority (N) where N is a number greater than zero that signifies the set of tests that will be built and consequently run.
```

Comment:
Can it be 0? I assume that would just be equivalent to not specifying a priority at all, but it is a valid option right?